docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,96 @@
 ---
 title: Tinylicious
 menuPosition: 1
-status: unwritten
+editor: sdeshpande3
 ---
 
 ## What is Tinylicious?
 
-Our local service used for prototyping
-
-Link to repo
+Tinylicious is a local, in-memory service used for prototyping and development purpose. You can find the code for the [Tinylicious service](https://github.com/microsoft/FluidFramework/tree/main/server/tinylicious) and [Tinylicious client](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/tinylicious-client). It can be instantiated against both `AzureClient` and `TinyliciousClient` for testing purpose.
 
 ## Using Tinylicious locally
 
+You can run tinylicious locally by executing the following command:
+
+```sh
+npx tinylicious@latest
+```
+
+By default, Tinylicious runs on port `7070`, however you can change port by setting an environment variable as `PORT` by running the below command in `Windows Powershell`,
+
+```sh
+$env:PORT=6502
+npx tinylicious@latest
+```
+
+Now, you can navigate to `http://localhost:6502` on your browser to see the Tinylicious service up and running.
+
 ## How to deploy using Tinylicious
 
+The `AzureClient` supports both instantiating against a deployed Azure Fluid Relay service instance for production scenarios, as well as against a local, in-memory service instance, known as Tinylicious, for development purposes.
+
+`AzureClient` can be connected to a local Tinylicious instance by passing in the tenant ID as `local`, the orderer and storage URLs  pointing to the Tinylicious instance on the default values of `http://localhost:7070`. You can use `InsecureTokenProvider` for token resolution while running the service locally for development purposes.
```

Comment:
When referring to usage with the `AzureClient`, lets refer to Tinylicious as Azure local service and have it run using `npx @fluidframework/azure-local-service@latest`